### ISSUE TITLE
Unable to select a project collection icon with the keyboard

### ISSUE THREAD

**Issue by zdavis** [ManifoldScholar/manifold#2034]
![image](https://user-images.githubusercontent.com/255909/54461555-236c2800-472a-11e9-9e58-d2592eb4e7ad.png)

**Comment by zdavis** [comment]
While you're in there, see about making a keyboard selection here as well:

![image](https://user-images.githubusercontent.com/255909/54461738-b73df400-472a-11e9-8bcd-4ca07e42f250.png)

**Comment by SMaxOwok** [comment]
> While you're in there, see about making a keyboard selection here as well:
> 
> ![image](https://user-images.githubusercontent.com/255909/54461738-b73df400-472a-11e9-8bcd-4ca07e42f250.png)

This one should be addressed in https://github.com/ManifoldScholar/manifold/commit/3af258a5eb9d6b31f958c46c98b237c6387c2fff

Will look into the icon selector though
